extras/help_generator/writers.py

```diff
@@ -148,26 +135,21 @@ def write_help_html(doc_dic, file, sli_command_list, keywords):
                                   title=name, footer=footertempl)
 
     if name:  # only, if there is a name
-        if file.endswith('.sli'):
-            f_file_name = open(('../cmds/sli/%s.html' % name), 'w')
-            f_file_name.write(cmdindexstring)
-            f_file_name.close()
-
-            f_file_name_hlp = open(('../cmds/sli/%s.hlp' % name), 'w')
-            f_file_name_hlp.write('\n'.join(hlplist))
-            f_file_name_hlp.close()
+        if fname.endswith('.sli'):
+            path = os.path.join(helpdir, 'sli')
         else:
-            f_file_name = open(('../cmds/cc/%s.html' % name), 'w')
-            f_file_name.write(cmdindexstring)
-            f_file_name.close()
+            path = os.path.join(helpdir, 'cc')
+
+        f_file_name = open((path + '/%s.html' % name), 'w')
+        f_file_name.write(cmdindexstring)
+        f_file_name.close()
 
-            f_file_name_hlp = open(('../cmds/cc/%s.hlp' % name), 'w')
-            f_file_name_hlp.write('\n'.join(hlplist))
-            f_file_name_hlp.close()
-        # return name
+        f_file_name_hlp = open((path + '/%s.hlp' % name), 'w')
```

Comment:
And here: Isn't this deprecated? Shouldn't we use ``"{}/{}.html".format(path, name)``?